audiobook/src/test/java/de/ph1b/audiobook/features/chapterReader/BinaryStreamExtensionsTest.kt

```diff
@@ -0,0 +1,88 @@
+package de.ph1b.audiobook.features.chapterReader
+
+import org.assertj.core.api.Assertions.*
+import org.bouncycastle.util.encoders.Hex
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.io.ByteArrayInputStream
+import java.io.EOFException
+
+@RunWith(RobolectricTestRunner::class)
+class BinaryStreamExtensionsTest {
+  private val binaryData = Hex.decode("6c6f6cffd7904df58a9e4e0c2582390eb2")
+  private var stream = ByteArrayInputStream(ByteArray(0))
```

Comment:
Thanks. :) You are writing quite idiomatically (especially as it's your first kotlin project).